app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,53 @@
+@import "foundation_and_overrides";
+$black: #000;
+$white: #ffffff;
+
+body {
+  background: url("Home_photo.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+
+.footer {
+  background: transparent;
+  font-size: 1.2rem;
```

Comment:
Properties should be ordered background, bottom, font-size, padding, position, text-align, width